### ISSUE TITLE
Healthcheck modification on ELB doesn't get applied, but stdout log shows it does.

### ISSUE THREAD

**Issue by jayasimhan** [hashicorp/terraform#446]
I had to change the health check port on a loadbalancer. I made the change to the script and ran terraform apply, console showed the following
```
aws_elb.cosnumer-api-router-elb: Modifying...
  health_check.0.target:        "http:8080/business/130002180" => "http:8009/business/130002180"
  listener.0.instance_protocol: "HTTP" => "http"
  listener.0.lb_protocol:       "HTTP" => "http"
```
But when I go in and check the ELB config in AWS, the change is not there.
```
Load balancer: cosnumer-api-router-elb
 
Ping Target	http:8080/business/130002180
Timeout	10 seconds
Interval	30 seconds
Unhealthy Threshold	2
Healthy Threshold	2

```

Let me know if I can help in anyway. Thanks.

**Comment by eukaryote** [comment]
This is still a bug. I just changed the target from "TCP:22" to "HTTP:80/" on an ELB, and Terraform incorrectly reported the change applied as:

```health_check.123456789.target:              "" => "HTTP:80/"```

It should have reported changing it from "TCP:22" to "HTTP:80/", and it didn't actually apply the change, even though Terraform reported "module.main.aws_elb.myelb: Modifications complete".

**Comment by catsby** [comment]
Sorry for the silence here – this is fixed on the `master` branch, and may also be fixed in release 0.3.7. 
I'm going to go ahead and close this, please let us know if you need anything else.
